keycloak-jboss/Dockerfile

```diff
@@ -20,3 +20,10 @@ ADD --chown=jboss:root ./standalone/ ./standalone
 
 RUN cp -R ./standalone/deployments/* /opt/jboss/keycloak/standalone/deployments
 
+#ARG container_user=mosip
+#ARG container_user_group=mosip
+#ARG container_user_uid=1001
+#ARG container_user_gid=1001
+#WORKDIR /home/${container_user}
+#RUN chown -R ${container_user}:${container_user} /home/${container_user}
+#USER ${container_user_uid}:${container_user_gid}
```

Comment:
Please remove this from here as we are not using JBoss keycloak

Reply:
we are not using jboss